app/models/expense.rb

```diff
@@ -0,0 +1,5 @@
+class Expense < ActiveRecord::Base 
+    validates :name, :amount, :hideout_id, presence: {strict: true}
```

Comment:
Rails seems to do this automatically. Is there a reason we need to be explicit here?

Reply:
This was needed with the permit to check for non-blank values, but not needed otherwise when ausing a require call